Assets/00_Content/Scripts/Rounds/ScoreCard.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class ScoreCard : MonoBehaviour {
+	[SerializeField] private GameObject scoreCardUI;
```

Comment:
This seems to be referring to itself, can't you use `gameobject` instead?

Reply:
Fixed